addon/components/select-input.js

```diff
@@ -0,0 +1,42 @@
+import Input from './abstract-input'
+import * as listUtils from '../list-utils'
+import Ember from 'ember'
+
+export default Input.extend({
+  classNames: [
+    'frost-bunsen-input-select',
+    'frost-field'
+  ],
+
+  init () {
+    this._super(...arguments)
+    this.set('options', Ember.A([]))
+  },
```

Comment:
Instead of overriding `init` here just do the following since `Input` uses the `PropTypeMixin` from `ember-prop-types`:
```javascript
getDefaultProps () {
  return {
    options: Ember.A([])
  }
},
```